test/e2e/case17_policy_webhook_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"open-cluster-management.io/governance-policy-propagator/test/utils"
+)
+
+const (
+	case17PolicyLongYaml       string = "../resources/case17_policy_webhook/case17_policy_long.yaml"
+	case17PolicyReplicatedYaml string = "../resources/case17_policy_webhook/" +
+		"case17_policy_replicate.yaml"
+	longNamesapce              string = "long-long-long-long-long-long-long"
+	longRepliNamesapce         string = "long-long-long-long-long-long-long-replica"
+	case17PolicyReplicatedName string = "case17-test-policy-replicated"
+	errMsg                     string = `admission webhook "policy.open-cluster-management.io.webhook" denied the ` +
+		`request: the combined length of the policy namespace and name <namespace>.<name> ` +
+		`should not exceed 63 characters`
+)
+
+var _ = Describe("Test policy webhook", Label("webhook"), Ordered, func() {
+	BeforeAll(func() {
+		_, err := utils.KubectlWithOutput("create",
+			"ns", longNamesapce,
+		)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		_, err = utils.KubectlWithOutput("create",
+			"ns", longRepliNamesapce,
+		)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		_, err = utils.KubectlWithOutput("apply",
+			"-f", "../resources/case17_policy_webhook/"+
+				"long-name-cluster.yaml",
```

Comment:
The cluster name does not impact the validating webhook since the replicated policy is not evaluated, only the root policy.

Reply:
So can you just use the normal managed cluster that is part of the E2E tests instead of this long-name-cluster?

Reply:
you mean like 'managed1' ?

Reply:
Yes, that sounds good.

Reply:
`case17PolicyReplicatedYaml` has name: `case17-test-policy-replicated`, namepace:`  policy-propagator-test`. I tested this because I wanted to see that the policy is created even though it is copied in long cluster name. name + namespace(long cluster name) > 63